spec/integration/unhandled_errors_in_receivers_spec.rb

```diff
@@ -5,6 +5,21 @@
 require_relative "support/logging_receiver"
 require_relative "support/integration_test_helpers"
 
+class PoorlyBehavingErrorHandler < Pwwka::ErrorHandlers::BaseErrorHandler
+	def handle_error(receiver,queue_name,payload,delivery_info,exception)
```

Comment:
please look into unconventional indentation. should use soft tabs with two spaces